### PR TITLE
Add a dummy first CHANGELOG entry for future tracking

### PR DESCRIPTION
**Pull request by TheBlueMatt**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+0.0.98 should be considered a release candidate to the first alpha release of
+rust-lightning and the broader LDK. It represents several years of work
+designing and fine-tuning a flexible API for integrating lightning into any
+application. As lightning continues to evolve, and new use-cases for lightning
+develop, the API of LDK will continue to change and expand. However, starting
+with version 0.1, objects serialized with prior versions will be readable with
+the latest LDK. While rust-lightning is approaching the 0.1 milestone, language
+bindings components of LDK available at https://github.com/lightningdevkit are
+still of varying quality. Some are also approaching an 0.1 release, while
+others are still much more experimental.
```
Comment (ariard) [inline CHANGELOG.md:12]: "This release (0.0.98) isn't considered ready yet for mainet usage" ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# 0.0.98 - 2021-06-11
+
+0.0.98 should be considered a release candidate to the first alpha release of
+rust-lightning and the broader LDK. It represents several years of work
+designing and fine-tuning a flexible API for integrating lightning into any
+application. As lightning continues to evolve, and new use-cases for lightning
```
Comment (ariard) [inline CHANGELOG.md:6]: "or serves as a foundation for a written-from-scratch LN node" ?

**Review by ariard — COMMENTED** [review]
Otherwise LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# 0.0.98 - 2021-06-11
+
+0.0.98 should be considered a release candidate to the first alpha release of
+rust-lightning and the broader LDK. It represents several years of work
```
Comment (valentinewallace) [inline CHANGELOG.md:4]: I think we use "Rust-Lightning" capitalization elsewhere in the codebase. Fine with either but consistency would be nice.

**Review by valentinewallace — COMMENTED** [review]
Fine to approve after the current comments are addressed

**Comment by TheBlueMatt** [comment]
Addressed feedback.

**Review by valentinewallace — APPROVED** [review]

**Comment by ariard** [comment]
ACK 2c80810

**Comment by TheBlueMatt** [comment]
Squashed with no changes, will merge after CI:
```
$ git diff-tree -U1 2c808105 66726fb5
$
```

**Pull request merged**